public/app/features/dashboard-scene/edit-pane/DashboardEditPaneSplitter.tsx

```diff
@@ -178,7 +180,7 @@ function getStyles(theme: GrafanaTheme2, headerHeight: number) {
         position: 'sticky',
         zIndex: theme.zIndex.activePanel,
         background: theme.colors.background.canvas,
-        top: headerHeight,
+        top: isExtensionSidebarOpen ? '0px' : headerHeight,
```

Comment:
not sure I understand this, why does having the sidebar open affect this div?

Reply:
It is because the scrolling behaviour changes. If the sidebar is open, the pane/div that the splitter is part of scrolls, so we need to stick to `top: 0px`:
![image](https://github.com/user-attachments/assets/44b821c8-8f46-41d8-943c-c1c5131d66c1)

If the sidebar is closed, the main body scrolls. Since the body also includes the menu and top part, we need `top: headerHeight`:
![image](https://github.com/user-attachments/assets/2d0d941d-ed02-45e3-a331-992529ae07c6)

Reply:
@svennergr I see, maybe this logic needs to be part of useChromeHeaderHeight ? 

as it data trails is also using it the same way (fixed controls).

Reply:
@torkelo makes much more sense!

Fixed in 7670f28d236b5dcad4a0e115c15999d45453910b